src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTestData.cs

```diff
@@ -653,6 +653,8 @@ private static IEnumerable<object[]> SerializableObjects()
 
 #if netcoreapp
+            var ambiguousImplementationException = new AmbiguousImplementationException();
```

Comment:
We should have passed a message and base exception to it as in other examples but I don't think we need to change it right now.